.gitignore

```diff
@@ -9,6 +9,7 @@
 *.egg
 *.egg-info
 dist
+dist_cext
```

Comment:
Hi @lyw07 !

Wondering.. Why does it have its own top-level `kolibri/dist_cext` instead of living in `kolibri/dist/cext/` ?

It would be nice to keep the folder structure clean - if possible - such that devs can easily grasp it. For reference, there has been an attempt to describe the structure here: http://kolibri.readthedocs.io/en/master/dev/files.html

It exemplifies both that it's hard to document, but essentially that it's hard for us to understand what's going on by shear intuition :)